expenses-manager-frontend/src/main/frontend/src/styles.css

```diff
@@ -1,13 +1,19 @@
+@import "~bootstrap/dist/css/bootstrap.css";
+@import "~bootstrap-icons/font/bootstrap-icons.css";
+
+
 html, body {   
-    width: 100%;
-    height: 100%;
+    inline-size: 100%;
+    block-size: 100%;
     margin: 0;
-    font-family: "Outlet"; 
+    font-family: "Outfit"; 
```

Comment:
Aici problema cu fontul care nu se aplica o putem fixa daca adaugam:
```
.mat-typography {
    font-family: 'Outfit', sans-serif !important;
}
```
In styles.css.